stl/inc/xutility

```diff
@@ -2418,6 +2474,62 @@ namespace ranges {
         inline constexpr _End::_Cpo end;
     }
 
+    // CUSTOMIZATION POINT OBJECT ranges::_Uend
+    namespace _Unchecked_end {
+        // clang-format off
+        template <class _Ty>
+        concept _Has_member = _Unchecked_begin::_Has_member<_Ty> && requires(_Ty& __t) {
+            { __t._Unchecked_end() } -> sentinel_for<decltype(__t._Unchecked_begin())>;

```

Comment:
```suggestion
            __t._Unchecked_begin();
            { __t._Unchecked_end() } -> sentinel_for<decltype(__t._Unchecked_begin())>;
```

Reply:
Should `__t._Unchecked_begin();` be commented as being repeated for improved diagnostics?

Reply:
Yes, good idea.